README.md

```diff
@@ -297,22 +297,22 @@ A list of available JMX attributes can be found [here](https://graphdb.ontotext.
 
 GraphDB's Helm chart supports deploying GraphDB with or without security. This can be toggled through `graphdb.security.enabled`.
 If it is deployed with security enabled, a special provisioning user is used for repository provisioning, cluster linking, health checks and so on.
-Additional users can be added through the settings file: `files/config/settings.js`. The users are described with their roles, username and a bcrypt64 password.
+Additional users can be added through the users file: `files/config/users.js`. The users are described with their roles, username and a bcrypt64 password.
 
-The file is provisioned before GraphDB's startup with the configmap `graphdb.masters.settingsConfigmap`.
-It can be overridden with other configmap containing the `settings.js` file. The same configmap is used for the `graphdb.properties` file as well.
+The file is provisioned before GraphDB's startup with the configmap `usersConfigMap`.
+It can be overridden with other configmap containing the `users.js` file.
```

Comment:
can you add a few lines about settings.js and update the readme to reflect the changes on logback.xml, joloki-access and graphdb.properties?

Reply:
Added little information about all the config files that can be setting using a configMap